e2e/rbd.go

```diff
@@ -791,6 +791,112 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("test RBD volume encryption with user secrets based SecretsMetadataKMS", func() {
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass with error %v", err)
```

Comment:
You can modify the log format as : `e2elog.Failf("failed to delete storageclass: %v", err)`

Reply:
Change the same elsewhere too.

Reply:
I prefer to keep it the same, as in all other places in our e2e code.

Reply:
nit: I think we have quite a mix of formatting in the e2e code. Mostly we moved to the format Yati mentioned for new logs (it also matches what we have in the non-e2e code).

Reply:
Modified, will keep it in mind from next time :+1:  :wink: